app/mailers/asynchronous_notify.rb

```diff
@@ -0,0 +1,36 @@
+require 'celluloid'
+
+class AsynchronousNotify < ActionMailer::Base
```

Comment:
Yup, I get it ;-) Using sidekiq straight off is not quite like the rest of Gitlab... I'll see what I can do reasonably quickly...